api/models/app.go

```diff
@@ -82,6 +86,19 @@ func GetApp(name string) (*App, error) {
 
 var regexValidAppName = regexp.MustCompile(`\A[a-zA-Z][-a-zA-Z0-9]{3,29}\z`)
 
+func (a *App) StackName() string {
+	if a.Tags == nil {
+		return shortNameToStackName(a.Name)
+	}
+
+	nameTag, ok := a.Tags["Name"]
```

Comment:
More idiomatic Go is to put the if and assignment on the same line, and to do the positive condition first to limit the scope of the name variable.

```go
if name, ok := a.Tags["Name"]; ok {
  return shortNameToStackName(name) 
}

return a.Name
```